data/sql/updates/pending_db_world/tk-immunities-fix.sql

```diff
@@ -0,0 +1,3 @@
+-- 20033 and 20034 were 1|2048 and 20047  was 2048
+UPDATE `creature_template` SET `mechanic_immune_mask` = 0 WHERE `entry` IN (20033, 20034);
+UPDATE `creature_template` SET `mechanic_immune_mask` = 256|33554432 WHERE `entry` = 20047;
```

Comment:
```UPDATE `creature_template` SET `mechanic_immune_mask` = `mechanic_immune_mask` |256|33554432 WHERE `entry` = 20047;

Reply:
In this case no, technically I should have written it as remove 2048 add the others